docs/sources/source/asr/8kHz_models.rst

```diff
@@ -0,0 +1,39 @@
+8kHz Models
```

Comment:
Can you make a reference to your conversion script somewhere in here?

Reply:
Done, on line 11.